components/user/card-seekbar.vue

```diff
@@ -23,4 +23,5 @@ const currentDateTime = useNow();
 const currentPosition = computed(() => formatTime(props.startTimeMs, currentDateTime.value.getTime()));
 const songDuration = computed(() => formatTime(props.startTimeMs, props.endTimeMs));
 const percentage = computed(() => ((currentDateTime.value.getTime() - props.startTimeMs) / (props.endTimeMs - props.startTimeMs)) * 100);
+const clampedPercentage = computed(() => Math.min(Math.max(percentage.value, 0), 100));
```

Comment:
It doesn't work. Being honest with you I don't know why the percentage doesn't update correctly, it's always stuck at the starting position.

Checking with Vue devtools it says that it is a computed property, so it should

My code:

```typescript
const currentPosition = computed(() => formatTime(props.startTimeMs, currentDateTime.value.getTime()));
const songDuration = computed(() => formatTime(props.startTimeMs, props.endTimeMs));
const useProjector = createProjection([props.startTimeMs, props.endTimeMs], [0, 100]);
const percentage = useClamp(useProjector(currentDateTime.value.getTime()), 0, 100);
```

Reply:
That's because `currentDateTime.value.getTime()` isn't reactive, to fix it, simply make the parameter a function, like:
```ts
const useProjector = createProjection([props.startTimeMs, props.endTimeMs], [0, 100]);
const percentage = useClamp(useProjector(() => currentDateTime.value.getTime()), 0, 100);
```

Reply:
The percentage bar is moving now but it goes out of sync if I seek ahead/behind on Spotify, or if I change to a different track.

Reply:
I'm sorry. I wrapped the projection in a fuction as well and now it seems to be working

Reply:
Oh, that's fixed with *another* arrow function 😅 

```ts
const useProjector = createProjection(() => [props.startTimeMs, props.endTimeMs], [0, 100]);
const percentage = useClamp(useProjector(() => currentDateTime.value.getTime()), 0, 100);
```

Reply:
Oops, GitHub didn't refresh the comment, yeah, that was the fix :)

Reply:
Thanks for your patience! I'll remind this from now on if I'm facing the same issue